frontend/src/page/Fader/RGBFader.tsx

```diff
@@ -0,0 +1,62 @@
+import {FaderItem, RGBItem} from "../../model/BackendConnection";
+import {Fader} from "./Fader";
+import React, {useState} from "react";
+import {RGBMixerRectangle} from "../RGBMixerPicture/RGBMixerPicture";
+import {postSingleFader} from "../../controller/Fetching";
+
+function CreateRGBFaderItem(channeloffset:number, faderItem:FaderItem ) :FaderItem {
+    let newfaderItem:FaderItem = { channel:faderItem.channel + channeloffset, value:faderItem.value, type:faderItem.type, universe:faderItem.universe }
+    return newfaderItem;
+}
+
+export function CreateRGBItem( color:string, rgbItem:RGBItem, value:number) :RGBItem {
```

Comment:
Function is called CreateRGBItem but it needs an rgbItem as parameter to create one?

Reply:
Stimmt das ist ein Update mit Return. Werde ich umbenennen.